src/main/java/org/cactoos/set/Sorted.java

```diff
@@ -35,11 +36,14 @@
  *
  * @param <T> Set type
  * @since 1.0.0
- * @todo #1292:30min This class should also implements SortedSet
- *  from the java collection framework by delegating to the
- *  wrapped set. Some tests must be added for it.
  */
-public final class Sorted<T> extends SetEnvelope<T> {
+@SuppressWarnings("PMD.OnlyOneConstructorShouldDoInitialization")
```

Comment:
@vzurauskas this can be removed if you introduce the `SortedSetEnvelope` as proposer in the other comment

Reply:
@victornoel Right.